Sources/Clappr/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -319,7 +320,13 @@ open class AVFoundationPlayback: Playback {
     }
     
     @objc func onAccessLogEntry(notification: NSNotification?) {
+        updateNumberOfDroppedFrames()
```

Comment:
Since we have a variable called numberOfDropped frames this function name can be misleading, can we use something like updateDroppedFrames or addUpDroppedFrames?

Reply:
Done ;)